source/extensions/filters/http/aws_request_signing/config.cc

```diff
@@ -18,6 +18,8 @@ Http::FilterFactoryCb AwsRequestSigningFilterFactory::createFilterFactoryFromPro
     const envoy::extensions::filters::http::aws_request_signing::v3::AwsRequestSigning& config,
     const std::string& stats_prefix, Server::Configuration::FactoryContext& context) {
 
+  // TODO(rgs1): we should allow this to be globally disabled.
```

Comment:
by 'this' are you referring to the filter? or the credentials provider?

Reply:
The filter; e.g.: for the use case in which you only want it to be enabled for a few vhosts/routes (as opposed to globally).